docs/DeveloperGuide.adoc

```diff
@@ -806,9 +806,59 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |sort contacts based on their locality |better match students to tutors
+
+|`* * *` |user |edit the information stored |keep the information up to date
+
+|`* * *` |user |tag tutors and students with multiple values |
+
+|`* * *` |user |add contacts of both tutors and students |
+
+|`* * *` |user |categorise contacts between students and tutors |
+
+|`* * *` |user |group clients based on the subject they wish to study/teach |
+
+|`* * *` |user |record what preference the students/tutors have (male/female) |
+
+|`* * *` |user |see and compare the price that students/tutors are willing to pay/accept|match them based on pricing
+
+|`* * *` |user |have an autosave function |changes will be saved once i exit the application
+
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* *` |user |do multi-layer searching |narrow down the options
+
+|`* *` |student |request to know the qualification of the tutor |make my decision wisely
+
+|`* *` |tutor |request to know the grades of the students |make my decision wisely
+
+|`* *` |user |easily delete all students/tutors that quit with a single command |save time manually deleting
```

Comment:
Don't have to elaborate on the how (the part with `with a single command`), leave that part out so that the developer would not be constrained.

Reply:
updated